src/Stripe.net/Entities/StripeInvoice.cs

```diff
@@ -31,6 +34,8 @@ public class StripeInvoice : StripeEntityWithId, ISupportMetadata
         [JsonProperty("billing")]
         public StripeBilling? Billing { get; set; }
 
+        public string BillingReason { get; set; }
```

Comment:
minor but I almost created an Enum and then realized there are like 2 files today. We should discuss the benefits of Enums and whether it's worth porting all the properties to Enums (which I would happily do if so)